src/server/project.js

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var fs = require('fs'),
+    path = require('path'),
+    cordovaServe = require('cordova-serve'),
+    Q = require('q'),
+    dirs = require('./dirs'),
+    pluginUtil = require('./utils/plugins'),
+    prepareUtil = require('./utils/prepare'),
+    utils = require('./utils/jsUtils');
+
+/**
+ * @constructor
+ */
+function Project(simulator, platform) {
+    this._simulator = simulator;
```

Comment:
`simulatorProxy` is an object that provides the required interface for getting the state of the simulation. So yes, it is more like a context object, if by context we can also mean to exposing functions to get more information and also functions that could change the behaviour of the `simulator` instance. I don't have a clear example for the latest. But if `simulatorContext` give us a better understanding of the meaning of this object, then let's change it!